### PR TITLE
Downgrade image minimizer to 3.2.1

### PR DESCRIPTION
**Pull request by darylldoyle**
Related Issue/RFC: #167

### Description of the Change

Squoosh and the `image-minimizer-webpack-plugin` seem to have a memory leak that's causing builds to fail when running on CircleCI.

Downgrading the `image-minimizer-webpack-plugin` to version `3.2.1` fixes this, as per this comment: https://github.com/webpack-contrib/image-minimizer-webpack-plugin/issues/287#issuecomment-1004299057

### Alternate Designs

Just ask people to override the package when they're using CircleCI

```json
"overrides": {
  "10up-toolkit": {
    "image-minimizer-webpack-plugin": "3.2.1"
  }
},
```

### Possible Drawbacks

We're fixing the package version and will need to actively monitor it for updates as well as checking if those updates cause a regression in the memory leak.

### Verification Process

Used an NPM override to test the package update on a failing CircleCI build.

### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://github.com/10up/10up-toolkit/blob/develop/CONTRIBUTING.md) document.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my change.
- [ ] All new and existing tests passed.
- [ ] I have added a changeset to my PR. See [**CONTRIBUTING**](https://github.com/10up/10up-toolkit/blob/develop/CONTRIBUTING.md) document for instructions

### CODE REVIEW

File: .changeset/rude-mugs-cry.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"10up-toolkit": minor
```
Comment (Antonio-Laguna) [inline .changeset/rude-mugs-cry.md]: We're fixing an issue here rather than adding a new feature

```suggestion
"10up-toolkit": patch
```

**Comment by Antonio-Laguna** [comment]
@darylldoyle thanks a lot, this is looking great. Can you take a look at my suggestion with the changeset? Other than that it looks good to me!

**Comment by darylldoyle** [comment]
Thanks @Antonio-Laguna. Changeset updated 🙂

**Review by Antonio-Laguna — APPROVED** [review]

File: packages/toolkit/package.json [hunk 2]
```diff
@@ -40,7 +40,7 @@
     "eslint-webpack-plugin": "^3.1.1",
     "html-webpack-plugin": "^5.5.0",
     "ignore-emit-webpack-plugin": "2.0.6",
-    "image-minimizer-webpack-plugin": "^3.2.3",
+    "image-minimizer-webpack-plugin": "3.2.1",
```
Comment (Antonio-Laguna) [inline packages/toolkit/package.json:43]: @darylldoyle sorry we didn't get back to this but I've been thinking more about this. I think it makes sense to leave the the `^` so most users can benefit from updates and if someone runs into the issue they can just _pin_ the dependency in their package.json without causing any NPM errors. 

This means we would need to add some documentation around this. Are you keen to do that or would you rather want to leave that out of the PR?

```suggestion
    "image-minimizer-webpack-plugin": "^3.2.1",
```

**Comment by nicholasio** [comment]
Closing in favor of #221

**Pull request closed without merging**